keps/sig-scheduling/4328-node-affinity-based-eviction/kep.yaml

```diff
@@ -0,0 +1,49 @@
+title: "Node Affinity Based Eviction" 
+kep-number: 4328
+authors:
+  - "@AxeZhan"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-scheduling
+  - sig-node
+status: implementable
+creation-date: 2023-12-13
+reviewers:
+  - "@alculquicondor"
+  - "@kannon92"
+  - "@ffromani"
+  - "@dchen1107"
+  - "@mrunalp"
+approvers:
+  - "@alculquicondor"
+  - "@dchen1107"
+  - "@mrunalp"
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.30"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: NodeAffinityEviction
```

Comment:
If this is really about _preemption_, let's change the name of the feature accordingly.

Reply:
I disagree that this should be named Preemption.

Reply:
OK, I'm convinced.